cypress/features/regression/table.feature

```diff
@@ -3,28 +3,28 @@ Feature: Table component
 
   @positive
   Scenario: Verify the pagination is visible
-    When I open default "Table" component in noIFrame with "table" json from "commonComponents" using "paginate" object name
+    When I open default "Table Test" component in noIFrame with "table" json from "commonComponents" using "paginate" object name
     Then pagination is visible
 
   @positive
   Scenario: I enable selectable
-    When I open default "Table" component in noIFrame with "table" json from "commonComponents" using "selectable" object name
+    When I open default "Table Test" component in noIFrame with "table" json from "commonComponents" using "selectable" object name
     Then rows are selectable
 
   @positive
   Scenario: I disable selectable
-    When I open default "Table" component in noIFrame with "table" json from "commonComponents" using "selectableFalse" object name
+    When I open default "Table Test" component in noIFrame with "table" json from "commonComponents" using "selectableFalse" object name
     Then rows are not selectable
```

Comment:
I think these tests can be removed and shown in chromatic @DlgSHi because they don't test the functionality just weather or not they exist

Reply:
yes, @Alec-Walker-Sage those TC could be converted into `chromatic`